netmiko/alcatel/alcatel_sros_ssh.py

```diff
@@ -1,70 +1,258 @@
-"""Alcatel-Lucent SROS support."""
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 - NOKIA Inc. All Rights Reserved.
+# Please read the associated COPYRIGHTS file for more details.
```

Comment:
The "All Rights Reserved" clause would need dropped (sounds like it is generally inconsistent with the MIT license)

https://opensource.stackexchange.com/questions/2121/mit-license-and-all-rights-reserved


This probably need to change to:

# Copyright (c) 2014 - 2019  Kirk Byers 
# Copyright (c) 2014 - 2019  Twin Bridges Technology
# Copyright (c) 2019 Nokia Inc.
# MIT License - See License file at https://github.com/ktbyers/netmiko/blob/develop/LICENSE

I think that is more proper for the MIT License.

Reply:
License info updated.